source/includes/fact-export-options.rst

```diff
@@ -0,0 +1,6 @@
+You can choose whether to include import statements, driver syntax,
```

Comment:
These includes are the beginning of step 3 of both procedures. I'd like to keep them as includes because the copy is the same throughout. If we need to make edits to this copy, it will only need to be done in one place instead of two.

Reply:
Done.